Data Model/mainProperties.js

```diff
@@ -0,0 +1,122 @@
+export const data =
+    [
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'jest to',
+                en: 'instance of'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'miejscowość',
+                en: 'locality'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'dekanat',
+                en: 'deanery'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
+            labels: {
+                pl: 'diecezja',
+                en: 'diocese'
+            },
+            descriptions: {
+                pl: null,
+                en: null
+            }
+        },
+        {
+            id: null,
+            type: 'property',
+            datatype: 'time',
+            labels: {
+                pl: 'data',
+                en: 'date'
+            },
+            descriptions: {
+                pl: null,
+                en: 'Date may be used to express temporal information at any level of granularity. Recommended best practice is to use an encoding scheme, such as the W3CDTF profile of ISO 8601 [W3CDTF].'
+            }
+        },
+        {
+            id: null,
+            type: 'item',
+            labels: {
+                pl: 'Parafia (Kościół katolicki ob. łacińskiego)',
+                en: 'Parish (Latin Church)',
+                de: 'Pfarre',
+                la: 'Parochia',
+                ru: 'парафия'
```

Comment:
@bszady I think that items we should create with labels beginning the capital letter, what do you think?

Reply:
@bernardyn I think we should be consistent and in line with ontohgis in this aspect. The spelling of upper and lower case letters has a meaning in different languages, e.g. German. Besides, the script for us here has to distinguish this as well.  
![image](https://github.com/ontohgis/wikikul/assets/21007174/c9aa8143-dc65-4080-82f3-59d06153c451)
![image](https://github.com/ontohgis/wikikul/assets/21007174/45f2ae05-4c92-46a6-b4d7-5abfb6153576)